recipes/bcl2fastq-nextseq/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: bcl2fastq-nextseq
+  version: "0.1.0"
+
+source:
+  git_url: https://github.com/brwnj/bcl2fastq.git
+  git_rev: add0bd6669ced224223529d2ae3a8931431f8857
+
+build:
+  number: 0
+  skip: True # [not py27]
+
+requirements:
+  build:
+    - python >=2.7,<3
```

Comment:
Usually, we don't restrict the Python version here, because this is already done by skip above. Also, in a previous conda version this resulted in having builds that are not pinned to the Python release. Can you check if the create package contains *_py27* in the file name? If yes, I am fine with this.